tests/unit/plugins/framework_specific/dvc/__snapshots__/test_writer_dvc.ambr

```diff
@@ -676,33 +676,46 @@
 ---
```

Comment:
Yes I ran it on everything. This is because I also prettified the DVC output here: https://github.com/LineaLabs/lineapy/blob/7e17ddbd072ce78cbf296a175799cc0b8ff69435/lineapy/plugins/dvc_pipeline_writer.py#L175 

The actual logic of the files havent changed (please verify as part of review) just the spacing and formatting.